PiwikTracker/Context.swift

```diff
@@ -0,0 +1,115 @@
+            case "iPhone6,1":    return "iPhone 5s (GSM)"
+            case "iPhone6,2":    return "iPhone 5s (Global)"
+            case "iPhone7,1":    return "iPhone 6 Plus"
+            case "iPhone7,2":    return "iPhone 6"
+            case "iPhone8,1":    return "iPhone 6s"
+            case "iPhone8,2":    return "iPhone 6s Plus"
+            case "iPhone8,4":    return "iPhone SE"
+            case "iPhone9,1":    return "iPhone 7 (GSM+CDMA)"
+            case "iPhone9,2":    return "iPhone 7 Plus (GSM+CDMA)"
+            case "iPhone9,3":    return "iPhone 7 (Global)"
+            case "iPhone9,4":    return "iPhone 7 Plus (Global)"
+                
+            // iPod
+            case "iPod1,1":      return "iPod Touch 1G"
+            case "iPod2,1":      return "iPod Touch 2G"
+            case "iPod3,1":      return "iPod Touch 3G"
+            case "iPod4,1":      return "iPod Touch 4G"
+            case "iPod5,1":      return "iPod Touch 5G"
+            case "iPod6,1":      return "iPod Touch 6G"
+                
+            // iPad
+            case "iPad1,1":      return "iPad 1"
+            case "iPad2,1":      return "iPad 2 (WiFi)"
+            case "iPad2,2":      return "iPad 2 (GSM)"
+            case "iPad2,3":      return "iPad 2 (CDMA)"
+            case "iPad2,4":      return "iPad 2 (WiFi)"
+            case "iPad2,5":      return "iPad Mini 1 (WiFi)"
+            case "iPad2,6":      return "iPad Mini 1 (GSM)"
+            case "iPad2,7":      return "iPad Mini 1 (GSM+CDMA)"
+            case "iPad3,1":      return "iPad 3 (WiFi)"
+            case "iPad3,2":      return "iPad 3 (GSM+CDMA)"
+            case "iPad3,3":      return "iPad 3 (GSM)"
+            case "iPad3,4":      return "iPad 4 (WiFi)"
+            case "iPad3,5":      return "iPad 4 (GSM)"
+            case "iPad3,6":      return "iPad 4 (GSM+CDMA)"
+            case "iPad4,1":      return "iPad Air 1 (WiFi)"
+            case "iPad4,2":      return "iPad Air 1 (Cellular)"
+            case "iPad4,3":      return "iPad Air"
+            case "iPad4,4":      return "iPad Mini 2 (WiFi)"
+            case "iPad4,5":      return "iPad Mini 2 (Cellular)"
+            case "iPad4,6":      return "iPad Mini 2 (Rev)"
+            case "iPad4,7":      return "iPad Mini 3 (WiFi)"
+            case "iPad4,8":      return "iPad Mini 3 (A1600)"
+            case "iPad4,9":      return "iPad Mini 3 (A1601)"
+            case "iPad5,1":      return "iPad Mini 4 (WiFi)"
+            case "iPad5,2":      return "iPad Mini 4 (Cellular)"
+            case "iPad5,3":      return "iPad Air 2 (WiFi)"
+            case "iPad5,4":      return "iPad Air 2 (Cellular)"
+            case "iPad6,3":      return "iPad Pro 9.7 (WiFi)"
+            case "iPad6,4":      return "iPad Pro 9.7 (Cellular)"
+            case "iPad6,7":      return "iPad Pro 12.9 (WiFi)"
+            case "iPad6,8":      return "iPad Pro 12.9 (Cellular)"
+                
```

Comment:
Is the Piwik SDK supposed to run on WatchOS and tvOS, too? Then we should add the proper device names here.